packages/core-mobile/app/services/fcm/FCMService.ts

```diff
@@ -63,9 +74,7 @@ class FCMService {
   listenForMessagesBackground = (): void => {
     messaging().setBackgroundMessageHandler(async remoteMessage => {
       Logger.info('A new FCM message arrived in background', remoteMessage)
-      const result = NotificationsBalanceChangeSchema.safeParse(
-        remoteMessage.data
-      )
+      const result = NotificationsBalanceChangeSchema.safeParse(remoteMessage)
```

Comment:
this seems to be a breaking change. has you worked with backend to make sure this change is backward compatible?

Reply:
this is actually bug, we should be passing `remoteMessage`, not `remoteMessage.data `since schema expects to find `data` and `notification` properties in passed object.